internal/controller/runtime/pod_client.go

```diff
@@ -36,7 +36,13 @@ func newPodClient(config *rest.Config) (rest.Interface, error) {
 // +kubebuilder:rbac:groups="",resources="pods/exec",verbs={create}
 
 func NewPodExecutor(config *rest.Config) (podExecutor, error) {
-	client, err := newPodClient(config)
+	// Create a copy of the config to avoid modifying the original
+	configCopy := rest.CopyConfig(config)
+
+	// Set unlimited QPS
+	configCopy.QPS = -1
```

Comment:
Since this configuration can be overwritten by setting a `RateLimiter flowcontrol.RateLimiter`, I propose we disable it by setting the `configCopy.RateLimiter`. e.g. we can do something like this

```
configCopy.RateLimiter = flowcontrol.NewFakeAlwaysRateLimiter()
```

WDYT?

Reply:
From the docs:

```
	// Rate limiter for limiting connections to the master from this client. If present overwrites QPS/Burst

```

Reply:
OK, this is done.